warning/warning-tests.ts

```diff
@@ -0,0 +1,13 @@
+import * as warning from "warning";
```

Comment:
I'm using es6 modules with typescript throughout my projects to have tree shaking support in webpack. Writing tests this way would ensure that I can use this project in a es6 environment. Various type definitions have been changed lately to support this style, e.g. https://github.com/DefinitelyTyped/DefinitelyTyped/blob/types-2.0/object-assign/object-assign-tests.ts. 

What is the policy of the DT team behind this? Has it changed lately?